pkg/cmd/cache/reset.go

```diff
@@ -24,25 +35,25 @@ func NewCmdResetCache() *cobra.Command {
 		Use:   "reset",
 		Short: "Reset the cache for a project",
 		Args:  cli.RequiresMaxArgs(1),
+
 		RunE: func(cmd *cobra.Command, args []string) error {
-			var cwd string
-			if len(args) > 0 {
-				cwd, _ = filepath.Abs(args[0])
+			cwd, err := filepath.Abs(args[0])
+			if err != nil {
+				return fmt.Errorf("unable to determine absolute path: %w", err)
```

Comment:
The logic from before isn't really easy to follow, sorry about that. What's happening is that `cache reset` can take either 0 or 1 arguments. If no arguments are specified, then the `cwd` variable will contain an empty string `""`.

Then if an empty string passed to `helpers.ResolveProjectID(projectID, cwd)` as the `cwd`, that function will actually resolve the current working directory.

So we do only want to try to resolve `args[0]` if `len(args) > 0`.

Maybe to make it clearer, we could rename the `cwd` variable here `cwdOverride` or `projectDir` or something that would make it more clear?

We could also move the resolution of `cwd, _ := filepath.Abs(...)` into the helper function itself, I imagine every command has that part duplicated currently.